docs/01-Using-Fleet/04-Adding-hosts.md

```diff
@@ -25,9 +26,22 @@ fleetctl package --type pkg --fleet-url=[YOUR FLEET URL] --enroll-secret=[YOUR E
 
 When you install the generated osquery installer on a host, this host will be automatically enrolled in the specified Fleet instance.
 
+## Signing installers
```

Comment:
```suggestion
### Signing installers
```
Can you please add a third `#` character here? This is because I think we'd like the "Signing installers" section to look like a subsection of the "Osquery installers" section.